docs/en/reference/configuration.rst

```diff
@@ -292,9 +292,6 @@ the corresponding platform class.
 You can also pass the ``serverVersion`` option if you want to disable automatic
 database platform detection and choose the platform version implementation explicitly.
 
```

Comment:
What I mean is that should we change this paragraph to something like the following rather than completely removing it:

```
The value of `serverVersion` option should be a real and valid version string, just like the response to a query of database version.

For example when using MySQL or MariaDB, it should be what returned by `SELECT VERSION();`,
like '10.9.3-MariaDB-1:10.9.3+maria~ubu2204'
```

Reply:
:thinking: if we add this to the docs, we should add it to 3.6.x IMO, or even on 3.5.x.

Reply:
Agreed, let's change the docs on 3.5.x already.